modules/installation-configuration-parameters.adoc

```diff
@@ -827,6 +849,33 @@ ifdef::rhv[]
 |`<machine-pool>.platform.ovirt.vmType`
 |Optional. The VM workload type, such as `high-performance`, `server`, or `desktop`.
 |String
+
+|`<machine-pool>.platform.ovirt.affinityGroupsNames`
+|Required. A list of affinity group names that should be applied to the VMs. The affinity groups must exist in {rh-virtualization}, or be created during installation as described in xref:Additional-virt-parameters-for-clusters[Additional {rh-virtualization} parameters for clusters]. This entry can be empty.
```

Comment:
Not required

Reply:
@stoobie ^^

Reply:
@Gal-Zaidman 

Fixed. It now says *Optional.*